src/exit_node.hpp

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 //*****************************************************************************
 #pragma once
-
+#include <memory>
```

Comment:
You do not use this in header, maybe move it to source file?

Reply:
I do:
std::unique_ptr<NodeSession> createNodeSession(const NodeSessionMetadata& metadata) override;